scripts/base_network.py

```diff
@@ -522,6 +534,18 @@ def base_network(
             axis=1,
             result_type="reduce",
         )
+        lines_ac.drop(
```

Comment:
Same as above

Reply:
Fixed in #1065: no need for the drop in `base_network` anymore.

Reply:
> Great! What would be your idea here, merge this and then 1065, merge 1065 here?

Have created [a PR on top](https://github.com/ekatef/pypsa-earth/pull/1) of #1065 which enables linopy in addition to the upgrade of PyPSA version